src/components/shared/TheInput.vue

```diff
@@ -13,11 +13,20 @@
     </div>
 </template>
 
-<script>
+<script lang="ts">
 export default {
-  name: 'TheInput',
-  props: ['width', 'modelValue', 'value', 'conditions', 'errorText'],
-  emits: ['update:modelValue'],
   inheritAttrs: false
 }
 </script>
+
+<script setup lang="ts">
+defineProps<{
+  width?: string,
+  modelValue?: string,
+  value?: string,
+  conditions?: boolean,
```

Comment:
if it's a single boolean than it's a condition

Reply:
I'm sorry but I didn't understand what you mean.